internal/packages/internal/packagekickstart/kickstart.go

```diff
@@ -22,22 +27,38 @@ type KickstartResult struct {
 	GroupKindsWithoutProbes []schema.GroupKind
 }
 
-func Kickstart(_ context.Context, pkgName string, objects []unstructured.Unstructured) (
+func Kickstart(
+	_ context.Context, pkgName string,
+	objects []unstructured.Unstructured,
+	paramFlags []string,
```

Comment:
Nit: I find a bit unclear the fact that this param, which is a slice of strings, is called `paramOpts` before this invocation all the way up but in this function it becomes `paramFlags` and `paramOpts` becomes the actual parsed object.

I would stick with `paramOpts` for the slice of strings and use a different name for the parsed objects (like `paramOptsObj` or `parsedOpts`), just for consistency.